web-frontend/src/pages/TransferCoins/SendCoins/SendRedirect.tsx

```diff
@@ -0,0 +1,51 @@
+import { useNavigate } from 'react-router-dom';
+import { useResource } from '../../../custom/api';
+import { AccountObject } from '../../../models/AccountModel';
+import { routeFor } from '../../../utils';
+import { useState } from 'react';
+import Intl from '../../../i18n/i18n';
+
+function Redirect() {
```

Comment:
Can we be more specific here!? Redirect to where!?

Reply:
I'll add it in the comments.
it redirects to the send coin page with the right nickname in the url